### ISSUE TITLE
FileUpload on IE11 :  button "input type file" doesn't work

### ISSUE THREAD

**Issue by laurentperroteau** [primefaces/primeng#3915]
**I'm submitting a ...** 
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
http://plnkr.co/edit/ZEiymmoAnssDaEVNxMM7?p=preview

**Current behavior**
Component FileUpload on IE11 : works fine using drag n drop but doesn't work at all with the button "input type file". 

I tried the project after closed issue "FileUpload basic on Internet Explorer #3838", same problem.

**Expected behavior**

**Minimal reproduction of the problem with instructions** :
Using plunker on IE11, uploading a light image.

* **Angular version:** 4.26

* **PrimeNG version:** 4.2.0-rc.1

* **Browser:** [IE 11]

 
* **Language:** [TypeScript 2.0.3]

**Comment by veplechuc** [comment]
In my case the "Upload" button is always disabled (using IE 11, works fine with Chrome) I just run the plnkr code posted here on the IE 11 and is the same issue that I have.
Drag and Drop option works

**Comment by vanroda** [comment]
Fwiw, was already broken in version 4.1.3 and can be simulated on the PrimeNG FileUpload example

**Comment by veplechuc** [comment]
BTW, the component on the 4.1.2 version works.

**Comment by laurentperroteau** [comment]
New version 4.2.1 (and commit "Fixed IE11 issues with FileUpload" 54f277c3de491efd8584eb93acc2019afde73359) solves the problem.

Forked plunker with 4.2.1 : http://plnkr.co/edit/UUUAHHFE0aM4qYCebrTB?p=preview 4.2.1
